bittide/src/Bittide/ClockControl/Callisto.hs

```diff
@@ -161,7 +166,9 @@ callisto margin framesize targetCount updateEveryNCycles mask allDataCounts =
   b_k = D.fromSignal (_b_k <$> state)
 
   steadyStateTarget :: DSignal dom 0 Float
-  steadyStateTarget = D.fromSignal (_steadyStateTarget <$> state)
+  steadyStateTarget
```

Comment:
Using the `reframingEnalbed` flag at this point may cause problems, since `rfCheck` hasn't be applied yet and `steadyStateTarget` is only utilized on a state update. Thus `rfCheck` is able to overwrite the choice and keep it until the next update. If `_targetCorrection` gets updated during that time a reframe will still take place.

I'll cherry-pick this commit for #279 and propose a fix there.

Reply:
Thanks!